doc/LORE.md

```diff
@@ -0,0 +1,52 @@
+# Factions
+
+## Exodii
+
+### Lore owner: @I-am_erk
+
+| Faction synopis | Last Updated |
+| --- | ---|
+| Placeholder text, maybe one or two paragraphs | 06/15/2023 |
+| Faction Goals | 06/15/2023 |
+| Development Goals | 06/15/2023 |
+
+### Exodii Questions
+
+| Question | Answer | Last Update |
+| --- | ---| --- |
+| Can I join the Exodii? | Not at this time, though they will continue to provide quests as the player's relationship with them improves. | 06/15/2023 |
+| Why does Rubik talk like that? Why can't I talk to the quads? | Exodii are not a homogenous group with a single spoken language. Rubik happens to speak as close to modern English as any Exodii, though there are some phrases that don't translate well. | 06/15/2023 |
+| What do the Exodii eat? | --- | --- |
```

Comment:
I remember a discussion where it was suggested they scavenge and process food from wherever they land. Erk also has an idea for a "gardener" cyborg that'd ask you for seeds, suggesting that they also grow their own food. [ he made an md file for them over here](https://gist.github.com/I-am-Erk/4021cd688fb9229e067053886ae914ee)